docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM debian:bookworm
```

Comment:
Wow, really new!

Reply:
Then we could put binaries built by ourselves into the image for test. cc @wubx